src/ImageSharp.Web/Commands/PresetRequestParser.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.WebUtilities;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+
+namespace SixLabors.ImageSharp.Web.Commands
+{
+    /// <summary>
+    /// Parses preset name from the request querystring and returns the commands configured for that preset.
+    /// </summary>
+    public class PresetRequestParser : IRequestParser
```

Comment:
Let's rename this ` PresetOnlyQueryCollectionRequestParser`. Verbose but accurate. 😄

Reply:
I like accurate!